concourse/pipelines/guest-package-build.jsonnet

```diff
@@ -751,7 +751,7 @@ local buildpackageimagetask = {
                       '-zone=us-central1-a',
                       '-test_projects=oslogin-testing-project',
                       '-images=projects/gcp-guest/global/images/debian-10-((.:build-id)),projects/gcp-guest/global/images/debian-11-((.:build-id)),projects/gcp-guest/global/images/debian-12-((.:build-id)),projects/gcp-guest/global/images/centos-7-((.:build-id)),projects/gcp-guest/global/images/rhel-7-((.:build-id)),projects/gcp-guest/global/images/rhel-8-((.:build-id)),projects/gcp-guest/global/images/rhel-9-((.:build-id))',
-                      '-filter=(oslogin)|(storage_perf)',
+                      '-filter=(oslogin) -exclude=(storage_perf)',
```

Comment:
I think we can just remove this line. So it'll just look like `-filter=oslogin`. The reason being the filter already excludes all other tests except those that match the filter, so the exclude flag would be superfluous.

Reply:
will do, thanks.